### DISCUSSION TITLE
Support streaming via Google Cast protocol

### DISCUSSION THREAD

**Discussion by snuccio** [rern/rAudio#820] [Feature Requests]
Hi, I really enjoy rAudio-1 and I would like to use it to stream audio to chromecast, android tv and nest devices.

I haven't found any related discussion in the repo so posting this as a feature request.

**Comment by EarsofTin** [comment]
As long as we're asking for stuff, how about some new Beatles music. I haven't heard anything since Let It Be! 

All kidding aside, Chromecast capability would be a huge feature and maybe a huge difficulty to implement. I remember reading about some incompatibilities between Arch and C+ that prevented this. I do keep LMS on one of my Pi's so I can use its Chromecast Bridge plugin to cast to the porch or garage.

And how about an equalizer setting that makes AC/DC songs sound different from each other.  And maybe a Jerry Garcia hologram generator too. : )

**Comment by snuccio** [comment]
I get it's not a trivial integration, though my intent here was not just to request a super cool and infeasible feature but rather discussing and tracking its technical challenges and possible workarounds.

For example, I was thinking if using [BubbleUPnP](https://play.google.com/store/apps/details?id=com.bubblesoft.android.bubbleupnp&hl=en&gl=US) would be a good alternative, as it would allow casting from rAudio-1 working as DLNA server. I need to try and update this thread. Or maybe it could be integrated into the [rAudio Android app](https://play.google.com/store/apps/details?id=com.raudio)?
